src/meta/src/stream/stream_manager.rs

```diff
@@ -859,6 +859,45 @@ impl GlobalStreamManager {
         Ok(())
     }
 
+    pub(crate) async fn reschedule_fragment(
+        &self,
+        fragment_id: u32,
+        target: ReschedulePolicy,
```

Comment:
What's the behavior if the target parallelism is the same as the fragment was?

Reply:
This part should be skipped; I’ve refactored it.